llvm/lib/Target/RISCV/GISel/RISCVCallLowering.cpp

```diff
@@ -191,8 +198,7 @@ struct RISCVIncomingValueAssigner : public CallLowering::IncomingValueAssigner {
 
     if (RISCVAssignFn(DL, Subtarget.getTargetABI(), ValNo, ValVT, LocVT,
                       LocInfo, Flags, State, /*IsFixed=*/true, IsRet, Info.Ty,
-                      *Subtarget.getTargetLowering(),
-                      /*FirstMaskArgument=*/std::nullopt))
+                      *Subtarget.getTargetLowering(), RVVDispatcher))
```

Comment:
This needs a rebase now that https://github.com/llvm/llvm-project/pull/79343 has landed.